### ISSUE TITLE
Website is not cleaned up after rejecting the YT service

### ISSUE THREAD

**Issue by LukeWCS** [kiprotect/klaro#353]
Hi there

I have been working on the integration of Klaro in one of my websites (with phpBB) for a few days. I noticed that the website is not cleaned after the YT service is rejected. By cleaning, I mean that the YT videos are hidden again and the cookies and local storage are also cleaned up.

I then downloaded all versions back to 0.7.0 and tested them individually. I found out that 0.7.4 is the last version where the cleanup works. I could not test 0.7.5 and 0.7.6 because these versions cause the following error and are terminated prematurely:

`Klaro.js? assets_version = 526: formatted: 8019 Uncaught TypeError: Cannot read property 'attributes' of null`

This message is caused at 0.7.5.

From 0.7.7 Klaro works again for me. This means that there must have been a change in one of the versions 0.7.5, 0.7.6 or 0.7.7, which is responsible for the fact that the page is no longer cleaned if a service is later rejected.

* With versions from 0.7.0 up to and including 0.7.4, the cleanup works without any problems, without having to reload the page.
* The cleanup does not work with versions 0.7.7 up to and including 0.7.11.

**Comment by LukeWCS** [comment]
Further observations:

1. If at 0.7.11 the Klaro UI is called after deactivating the service (`Klaro.show`), the cleanup is carried out immediately, even before I perform an action in the Klaro UI.
1. When I make the following change, Klaro works as expected:

```
if (r.src === s) {
	console.debug("Skipping ".concat(r.tagName, " for service ").concat(e.name, ", as it already has the correct type..."));
	// continue
}
```

**Comment by adewes** [comment]
Hi @LukeWCS, we recently released a new version of the contextual consent mechanism which should address these issues, please let me know if they're still present.

**Comment by LukeWCS** [comment]
@adewes 

Hi Andreas

Yes, I had already tested 0.7.12 in February and it worked again from this version. Just after your message here, I tested 0.7.18 and this version also works as it should. That means, the problem is fixed with 0.7.12+.

Thanks :)
